moonstone/normalization/reads/read_downsize.py

```diff
@@ -0,0 +1,69 @@
+import logging
+import random
+
+from moonstone.normalization.reads.base import BaseDownsizing
+
+logger = logging.getLogger(__name__)
+
+
+class DownsizePair(BaseDownsizing):
+    """Normalization for the purposes of assessing diversity. Reads are downsized by random selection of raw reads
+    generating a subset from which alpha diversity can be calculated.
+    Note that removal of data, while useful for diversity assessment, is no longer considered good practice.
+    https://doi.org/10.1371/journal.pcbi.1003531
+    """
+
+    def __init__(self, raw_f, raw_r, n=1000, seed=623):
```

Comment:
Maybe you could give more precision that by `raw_f` and `raw_r` you actually mean the path of the files? I got confused, this is why I put this comment about memory previously.

Reply:
Exactly, these are the file paths. Consequently, there is really no problem with memory.
Two potential issues here:
- should these variables be renamed to make the code clearer (probably yes, I think)
- Does the structure make sense given that the overloaded downsize_pair module does not return anything?

Reply:
You could still return the file names of the two generated files to respect the structure?

Reply:
I tried to clarify the variable names, now called raw_file_f, to indicate they are files and not actual data.